src/wp-admin/js/inline-edit-post.js

```diff
@@ -1,44 +1,74 @@
 /* global inlineEditL10n, ajaxurl, typenow */
 window.wp = window.wp || {};
-
+/**
+ * An object that manages the quick edit and bulk edit windows for editing posts or pages.
+ *
+ * @namespace inlineEditPost
+ *
+ * @type {Object}
+ *
+ */
 var inlineEditPost;
 ( function( $, wp ) {
-inlineEditPost = {
 
+	inlineEditPost = {
+
+	/**
+	 * Initialises the inline and bulk post editor.
+	 * @memberof    inlineEditPost
+	 * @fires       inlineEditPost#init
+	 * @returns     void
+	 */
 	init : function(){
 		var t = this, qeRow = $('#inline-edit'), bulkRow = $('#bulk-edit');
 
+		/**
+		 * Set if the editor is created on the posts or page edit-page.
+		 * @type {string}
+		 */
 		t.type = $('table.widefat').hasClass('pages') ? 'page' : 'post';
```

Comment:
type is a property isn't yet documented. This should be in the `@property` attribute the inlineEditPost object.
Also t.what ( a couple lines down)